db/migrate/20241101152454_drop_benefits_intake_uuid_from_form_submissions.rb

```diff
@@ -0,0 +1,5 @@
+class DropBenefitsIntakeUuidFromFormSubmissions < ActiveRecord::Migration[7.1]
+  def change
+    safety_assured { remove_column :form_submissions, :benefits_intake_uuid, :uuid }
```

Comment:
Sorry @Thrillberg, I should have noticed this as well
`  safety_assured { remove_column :form_submissions, :benefits_intake_uuid, :uuid, if_exists: true }`

Reply:
No worries! Added!